packages/p2p/src/components/buy-sell/buy-table.jsx

```diff
@@ -44,7 +44,16 @@ export class BuyTable extends React.Component {
 
         if (api_error_message) return <TableError message={api_error_message} />;
 
-        return <InfiniteLoaderList items={items} RenderComponent={Row} RowLoader={BuySellRowLoader} />;
+        return (
+            <InfiniteLoaderList
+                // screen size - header size - footer size - page overlay header - page overlay content padding -
+                // tabs height - padding+margin of tab content - toggle height - table header height
+                initial_height={'calc(100vh - 48px - 36px - 41px - 2.4rem - 36px - 3.2rem - 40px - 52px)'}
```

Comment:
instead of this comments, you can make those into variables with the naming

Reply:
@mustofa-binary For this case, I don't think it's more readable to extract the numbers into variables. Also, we can't use SCSS variables here either, and we should ideally not create app-specific constants inside the P2P package.

P.S.: Currently the values are set according to Deriv header/footer size, etc. But we will test this in Binary to see how it looks.